package-lock.json

```diff
@@ -2517,20 +2517,6 @@
         "node": ">=8"
       }
     },
```

Comment:
bring this file back pls

Reply:
What do you mean? The file auto updates.